apps/selvbetjening-ui/src/common/knappMeny/KnappMeny.tsx

```diff
@@ -0,0 +1,61 @@
+import { BodyShort, Box, Button, HGrid, VStack } from '@navikt/ds-react'
+import { ArrowLeftIcon, ArrowRightIcon, FloppydiskIcon, PaperplaneIcon, TrashIcon } from '@navikt/aksel-icons'
+import { Navigate, useNavigate } from 'react-router-dom'
+import { useSpraak } from '../spraak/SpraakContext.tsx'
+import { useSanityInnhold } from '../sanity/useSanityInnhold.ts'
+import { FellesKomponenter } from '../../sanity.types.ts'
+
+interface Props {
+    tilbakePath: string
+    nestePath: string
+    skalSendeSoeknad?: boolean
+}
+
+export const KnappMeny = ({ tilbakePath, nestePath, skalSendeSoeknad }: Props) => {
```

Comment:
Kan være litt vanskelig å forstå hva KnappMeny gjør?

```suggestion
export const NavigasjonMeny = ({ tilbakePath, nestePath, skalSendeSoeknad }: Props) => {
```

Reply:
E9!